id: PYSEC-2020-43
package:
  name: flask-cors
  ecosystem: PyPI
summary: Vulnerability in flask-cors
details: An issue was discovered in Flask-CORS (aka CORS Middleware for Flask) before
  3.0.9. It allows ../ directory traversal to access private resources because resource
  matching does not ensure that pathnames are in a canonical format.
affects:
  ranges:
  - type: ECOSYSTEM
    fixed: 3.0.9
references:
- type: WEB
  url: https://github.com/corydolphin/flask-cors/releases/tag/3.0.9
- type: WEB
  url: http://lists.opensuse.org/opensuse-security-announce/2020-09/msg00028.html
- type: WEB
  url: http://lists.opensuse.org/opensuse-security-announce/2020-09/msg00032.html
- type: WEB
  url: http://lists.opensuse.org/opensuse-security-announce/2020-09/msg00039.html
- type: WEB
  url: http://lists.opensuse.org/opensuse-security-announce/2020-09/msg00048.html
- type: WEB
  url: https://www.debian.org/security/2020/dsa-4775
aliases:
- CVE-2020-25032
modified: "2020-10-21T12:15:00Z"
published: "2020-08-31T04:15:00Z"
